### PR TITLE
PMM-12887 Implemented new PG dashboard.

### PR DESCRIPTION
**Pull request by guriandoro**
New dashboard as showcased in https://www.percona.com/blog/postgresql-checkpoints-buffers-and-wal-usage-with-percona-monitoring-and-management/.

We don't need a custom exporter anymore, because we are using pg_stat_replication_pg_current_wal_lsn_bytes, which is being exported in the latest PMM client versions.

### CODE REVIEW

**Comment by rnovikovP** [comment]
Let's move this dashboard to the Experimental Folder first, so we can show it to more people while working on improving it.

**Review by rnovikovP — CHANGES_REQUESTED** [review]
targeted folder -> Experimental

**Comment by guriandoro** [comment]
Moved dashboard to Experimental directory

File: dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json [hunk 1]
```diff
@@ -0,0 +1,5731 @@
...
+          },
+          "refId": "A"
+        }
+      ],
+      "title": "Node",
+      "type": "text"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
```
Comment (YashSartanpara1) [inline dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json:248]: Please change such Datasources to 
`"datasource": "Metrics",`

There are multiple occurrences

File: dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json [hunk 2]
```diff
@@ -0,0 +1,5731 @@
...
+      },
+      "type": "graph",
+      "xaxis": {
+        "mode": "time",
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "$$hashKey": "object:636",
```
Comment (YashSartanpara1) [inline dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json:949]: Please remove all the hashkey objects

Comment (guriandoro) [reply dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json]: Hi Yash! Thanks for the review. Please check the new commits.

File: dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json (hunk 1 above)
Comment (guriandoro) [reply dashboards/Experimental/PostgreSQL_Checkpoints_Buffers_and_WAL_Usage.json]: Hi Yash! Thanks for the review. Please check the new commits.

**Review by rnovikovP — APPROVED** [review]

**Review by YashSartanpara1 — APPROVED** [review]

**Pull request merged**